core/src/main/java/bio/terra/pearl/core/service/kit/KitRequestDetails.java

```diff
@@ -0,0 +1,85 @@
+package bio.terra.pearl.core.service.kit;
+
+import bio.terra.pearl.core.model.kit.KitRequest;
+import bio.terra.pearl.core.model.kit.KitRequestStatus;
+import bio.terra.pearl.core.model.kit.KitType;
+import bio.terra.pearl.core.service.exception.internal.InternalServerException;
+import bio.terra.pearl.core.service.kit.pepper.PepperKit;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Holds set of kit request data that is relevant to users
+ */
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+@Slf4j
+public class KitRequestDetails {
+  private UUID id;
+  private Instant createdAt;
+  private KitType kitType;
```

Comment:
if this is for display purposes, could we just send kitTypeName?

Reply:
The use of KitType precedes these changes. 
I attempted to code this change, but we use this DTO for UI (`displayName`) and export (`name`). Those usages made sense so I decided not to make this change. Of course we could include both `displayName` and `name`, but it seems like letting clients deconstruct the class instance for a particular use makes sense.
Let me know if you see this differently.